### DISCUSSION TITLE
如何加入实习？

### DISCUSSION THREAD

**Discussion by ActivePeter** [WasmEdge/WasmEdge#1349] [Q&A] [answered]
在这里看到的
https://xw.qq.com/cmsid/20220310A0CKZT00
对rust比较感兴趣，想加入实习

**Comment by alabulei1** [comment, marked as answer]
@ActivePeter  Sorry for my late reply. Please send an introduction email to vivian@secondstate.io. Thanks.

Also you could start to work at WasmEdge via GSoC. Check out this issue #1227
